mailer.js

```diff
@@ -162,9 +169,11 @@ module.exports = function (log) {
     if (message.redirectTo) { query.redirectTo = message.redirectTo }
     if (message.resume) { query.resume = message.resume }
 
-    var link = this.verificationUrl + '?' + qs.stringify(query)
+    var link = this._generateUTMLink(this.verificationUrl, query, template, 'activate')
+    var alternativeLink = this._generateUTMLink(this.verificationUrl, query, template, 'activate-alternative')
+
     query.one_click = true
-    var oneClickLink = this.verificationUrl + '?' + qs.stringify(query)
+    var oneClickLink = this._generateUTMLink(this.verificationUrl, query, template, 'activate')
```

Comment:
Instead of passing in `activate` and `activate-alternative` and `support`, it seems like you could [create another map, this time of template name to context](https://github.com/mozilla/fxa-auth-mailer/pull/190/files#diff-3427c5e3d78d8d6550178ad46b9e0680R11). The last parameter to _generateUTMLink could then be a context suffix instead of the context itself. A map could be used to [implement the support link issue mentioned above](https://github.com/mozilla/fxa-auth-mailer/pull/190/files#r72398196).